packages/blockchain-wallet-v4-frontend/src/data/components/interest/sagas.ts

```diff
@@ -434,8 +435,20 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
           throw new Error('Missing deposit address')
         }
 
+        const hotWalletAddress = selectors.core.walletOptions
+          .getHotWalletAddresses(yield select(), Product.REWARDS)
+          .getOrFail('No extra data') as String
+        if (typeof hotWalletAddress !== 'string') {
+          throw new Error('Hot Wallet Address not located.')
```

Comment:
Old way of sending in the `if` and log, new way with replaced address and memo in the `else` branch